README.md

```diff
@@ -32,12 +32,46 @@ sudo ln -s /usr/lib/llvm-15/bin/clang{,++,-format,-tidy} /usr/local/bin
 
 Python 3 is required to automatically build and link unit tests. Python 3 comes preinstalled on most Linux distributions, and is available as a package on all others.
 
-# Getting started (using Clang)
+Although not essential, to aid performance, we recommend running the OS in the non GUI mode. This can be achieved by running:
+```
+sudo raspi-config
+```
+Selecting 1 System Options, S5 Boot / Auto Login, B2 Console Autologin. Reboot and enjoy the resource saving.
+
+## Hardware setup:
+The Codec Zero should be wired up with the microphone on the left input channel and the synth on the right input channel of the auxilliary input. The Headphones or speaker shoudld be wired to either or both channels of the auxilliary output.
+
+![Circuit diagram](./Circuit.png)
+
+For testing, a Sure SM58 was chosen as the microphone (using pin 2 and 3 on the XLR connector) and a software synthesizer on a pc was used as the synthesizer. A portable speaker was used as the output device. If a low sensitivity microphone is used it may need to be amplified with an external amplifier. The volume of the microphone can balanced with the synthesizer by changing the microphone gain argument when running the program.
```

Comment:
```suggestion
For testing, a Shure SM58 was chosen as the microphone (using pin 2 and 3 on the XLR connector) and a software synthesizer on a pc was used as the synthesizer. A portable speaker was used as the output device. If a low sensitivity microphone is used it may need to be amplified with an external amplifier. The volume of the microphone can balanced with the synthesizer by changing the microphone gain argument when running the program.
```

Reply:
I was going to ask that after you'd decided what your command line options were going to be

Reply:
Decided it's too late to fix it, so remove that section thnx